src-ml-test-app/source/debug-screens/list/ListEnterExitScreen.bs

```diff
@@ -433,15 +458,15 @@ class ListEnterExitScreen extends mv.BaseScreen
   end function
 
   private function onCellWillGainFocus(event as ml.ListEvent)
-    ? "+SCREEN onCellWillGainFocus : R: "; event.rowIndex ; " I: " ; event.cellIndex; " D:" ; event.direction
+    ' ? "+SCREEN onCellWillGainFocus : R: "; event.rowIndex ; " I: " ; event.cellIndex; " D:" ; event.direction
     mioc.setInstance("focusedIndex", {
```

Comment:
i'm thinking this was a part of an old experiment and it can be removed, to avoid confusions

Reply:
commented out. all of these are useful so I keep them around - they can quickly let me know if events are flowing through the entire list.